yarn.lock

```diff
@@ -17649,42 +17656,42 @@ __metadata:
   languageName: node
   linkType: hard
 
-"express@npm:^4.17.1, express@npm:^4.17.3":
-  version: 4.19.2
-  resolution: "express@npm:4.19.2"
+"express@npm:^4.17.3, express@npm:^4.20.0":
+  version: 4.21.0
+  resolution: "express@npm:4.21.0"
```

Comment:
Let's change the package.json version to 4.21.0, in practice it's resolved to that version anyway.